shared/chat/conversation/list-area/normal/index.stories.js

```diff
@@ -52,6 +55,53 @@ class Rnd {
 
+// Generate timestamp in a range between start and end with some
+// messagesThreshold number of consecutive messages with the same timestamp
+const makeTimestampGen = () => {
+  const r = new Rnd(1337)
+  const origin = {year: 2018, month: 0, day: 0}
+
+  let messagesThreshold: number = 0
+  let generatedCount: number = 0
+  let currentTimestamp: number = 0
+
+  let dayRange: number = 0
+  let start = moment(origin)
+  let end = moment(origin)
+
+  return (): number => {
+    // Initialize or reset because threshold was crossed
+    if (currentTimestamp === 0 || generatedCount > messagesThreshold) {
+      // Move the start day up by the previous number of days to avoid overlap
+      start.add(dayRange, 'days')
+      // Get a new date range for random timestamps
+      dayRange = r.next() % 7 + 1
+      end.add(dayRange, 'days')
+
+      const diff = end.diff(start)
+      // Multiply the epoch time different by some floating point between [0, 1]
+      const newDiff = diff * (r.next() / 2147483647)
+      const newTimestamp = moment(start.valueOf() + newDiff)
+      currentTimestamp = newTimestamp.valueOf()
+
+      // Reset threashold and count
+      messagesThreshold = r.next() % 10 + 1
```

Comment:
You could parameterize the `10` here and `7` on L78 to make it easily configurable. it might also make this fcn more self explanatory.